SimplifiedPublication.go

```diff
@@ -0,0 +1,118 @@
+package crossrefindexer
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+func GetSimpleTitle(pub *CrossRef) []string {
+	simpleTitle := pub.Title
+	if len(simpleTitle) == 0 {
+		return []string{""}
+	}
+	for pos, title := range simpleTitle {
+		title = strings.Replace(title, "\n", " ", -1)
+		title = strings.Replace(title, "( )+", " ", -1)
+		simpleTitle[pos] = strings.TrimSpace(title)
```

Comment:
Slices are pointers in Go, so you should be aware that you are mutating the original data here. Which is fine, but just to make sure that you internalize this so that you won't get bitten by this in the future